config/env.development

```diff
@@ -260,3 +263,54 @@ FEED_QUEUE_PARALLEL_WORKERS=1
 
 # Max number of posts per page
 MAX_POSTS_PER_PAGE=5
+
+
+################################################################################
+# Supabase Service
+################################################################################
+
+SUPABASE_API_URL=http://kong:8000
+
+# Secrets
+# YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+
+POSTGRES_PASSWORD=your-super-secret-and-long-postgres-password
+JWT_SECRET=your-super-secret-jwt-token-with-at-least-32-characters-long
+ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
+SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJzZXJ2aWNlX3JvbGUiLAogICAgImlzcyI6ICJzdXBhYmFzZS1kZW1vIiwKICAgICJpYXQiOiAxNjQxNzY5MjAwLAogICAgImV4cCI6IDE3OTk1MzU2MDAKfQ.DaYlNEoUrrEn2Ig7tqibS-PHK5vgusbcbo7X36XVt4Q
+
+# Auth
+
+## General
+
+// Telescope base URL
+SITE_URL=http://localhost:8000
+ADDITIONAL_REDIRECT_URLS=
+JWT_EXPIRY=3600
+DISABLE_SIGNUP=false
+
+## Email auth
+ENABLE_EMAIL_SIGNUP=true
```

Comment:
Should this be false?  We don't want email signup

Reply:
I'm not sure, don't we want to create a new user using their email?
https://supabase.com/docs/reference/javascript/auth-signup

Reply:
My (very limited) reading of this is that setting this to `true` means users can use their email to sign-up to Supabase.  We don't really want to enable sign-ups at all, since we want to manage them ourselves via the Telescope SSO service, right?  Or am I misunderstanding this.

Reply:
I see what you mean. When we get the user info from SSO and the registration flow, we need to register them with their email so next time they sign in we know this user is both a Seneca and Telescope user. If email sign-up is disabled, is there another way to sign up?

Reply:
Yeah, I'm not sure.  Let's leave it like this, and we'll test it out and make changes once we have it working and get tests written.  It's hard to reason about this based solely on this config.